inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongParser.java

```diff
@@ -109,6 +114,30 @@ public static InlongStreamInfo parseStreamInfo(Response response) {
             FullStreamResponse fullStreamResponse = GsonUtil.fromJson(fullStreamJson.toString(),
                     FullStreamResponse.class);
             list.add(fullStreamResponse);
+            //Parse sourceResponse in each stream
+            JsonArray sourceJsonArr = fullStreamJson.getAsJsonArray(SOURCE_INFO);
+            List<SourceResponse> sourceResponses = Lists.newArrayList();
+            fullStreamResponse.setSourceInfo(sourceResponses);
+            for (int j = 0; j < sourceJsonArr.size(); j++) {
+                JsonObject sourceJson = (JsonObject) sourceJsonArr.get(i);
+                String type = sourceJson.get(SOURCE_TYPE).getAsString();
+                SourceType sourceType = SourceType.forType(type);
+                switch (sourceType) {
+                    case BINLOG:
+                        BinlogSourceResponse hiveSinkResponse = GsonUtil.fromJson(sourceJson.toString(),
```

Comment:
Maybe this variable can name `binlogSourceResponse`?

Reply:
yes